client/src/panels/welcome_mat/welcome_mat.js

```diff
@@ -158,8 +158,8 @@ const WelcomeMat = (props) => {
     latest_hist_spend_data,
     oldest_hist_spend_data,
   } = calcs;
-
-  const actual_hist_years_apart = _.parseInt( _.split(latest_hist_spend_data.year, '-') ) - _.parseInt( _.split(oldest_hist_spend_data.year, '-') ) + 1;
+  const current_year = _.parseInt( _.split( _.last(actual_history_years), '-') ) + 1;
+  const actual_hist_years_apart = current_year - _.parseInt( _.split(oldest_hist_spend_data.year, '-') );
```

Comment:
This is weird because `_.split(...)` returns an array of length 2, so it's a bit surprising that `_.parseInt` returns just the parsed value of the first element. I'd prefer the more explicit
```
_.chain(oldest_hist_spend_data.year).split('-').head().parseInt().value()
```